src/lib/modernExtend.ts

```diff
@@ -28,6 +29,7 @@ const timeLookup = {
     '1_HOUR': 3600,
     '30_MINUTES': 1800,
     '10_SECONDS': 10,
+    ...constants.repInterval,
```

Comment:
I want to get rid of `repInterval`, can you move the used on to here?

Reply:
Done.